ext/opcache/tests/preload_loadable_classes_2.phpt

```diff
@@ -9,11 +9,17 @@ opcache.preload={PWD}/preload_loadable_classes_2.inc
 <?php
 require_once('skipif.inc');
 if (PHP_OS_FAMILY == 'Windows') die('skip Preloading is not supported on Windows');
-if (getenv('SKIP_ASAN')) die('xfail Startup failure leak');
 ?>
 --FILE--
-Unreachable
+<?php
```

Comment:
From a reader's perspective, this test is not really understandable: previously, it was clear that `--FILE--` was not being used, but now it is, which makes the change in expected results quite suspicious

Reply:
Previously, opcache couldn't preload classes with unresolved constants. Now this is not a problem. Constants that are not resolved during preloading, may be resolved later at run-time.